src/Translations.js

```diff
@@ -471,7 +471,7 @@ async function pointReferencesToTranslatedDocument(data, language) {
 
 async function pointToTranslatedDocument(reference, language) {
   const { _type, translationId } = await sanityClient.fetch(
-    groq`*[_id == $ref][0] { _type, translationId }`,
+    groq`*[_id == $ref || _id == 'drafts.' + $ref][0] { _type, translationId }`,
     { ref: reference._ref }
   )
```

Comment:
It's possible that this reference references a document that's prefixed with 'drafts.' and does not have a published version (if it was created inline).